lib/host_kafka.py

```diff
@@ -0,0 +1,48 @@
+import socket
+
+from app.config import Config
+from app.environment import RuntimeEnvironment
+from app.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def _check_connection(kafka_socket, server):
+    conn_check = 999
+    try:
+        host, port = server.split(":")
+        nw_addr = (host, int(port))
+        conn_check = kafka_socket.connect_ex(nw_addr)
```

Comment:
```suggestion
        maybe_error_code = kafka_socket.connect_ex(nw_addr)
        if maybe_error_code == 0:
           return True
```

makes things way more easy on the loop

Reply:
Done!